website/docs/r/ecs_task_definition.html.markdown

```diff
@@ -134,6 +156,11 @@ For more information, see [Cluster Query Language in the Amazon EC2 Container
 Service Developer
 Guide](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/cluster-query-language.html).
 
+#### Proxy Configuration Arguments
+
+* `type` - (Optional) The proxy type. The only supported value is `APPMESH`.
```

Comment:
Nit: This argument should be moved below the required arguments since its optional and also mention that it defaults to `APPMESH`